Sources/Shared/Models/VIMVideo.h

```diff
@@ -94,6 +95,7 @@ typedef NS_ENUM(NSUInteger, VIMVideoProcessingStatus) {
 @property (nonatomic, strong, nullable) VIMReviewPage *reviewPage;
 @property (nonatomic, strong, nullable) VIMUpload *upload;
 @property (nonatomic, strong, nullable) FileTransfer *fileTransfer;
+@property (nonatomic, strong, nullable) APIClientApp *clientApp;
```

Comment:
Should we actually say `sourceClientApp` here, since in the context of `VIMVideo` it is the `source` API client app

Reply:
+1